.rubocop_todo.yml

```diff
@@ -66,9 +66,7 @@ RSpec/NamedSubject:
 # Configuration parameters: MaxNesting.
 RSpec/NestedGroups:
   Exclude:
-    - 'spec/unit/ability_spec.rb'
-    - 'spec/unit/catalog_spec.rb'
-    - 'spec/unit/enforcement_spec.rb'
+    - 'spec/**/*_spec.rb'
```

Comment:
Because I don't really care how deeply our groups are nested.  Actually, given that this is `RSpec/NestedGroups` maybe we should just turn it off?  See https://github.com/samvera/hyrax/blob/77140b38907158aaa4e30aae442ea5f04d0b1be2/.rubocop.yml#L156